### ISSUE TITLE
Text needs to be re-written with some clarity

### ISSUE THREAD

**Issue by neomatrix369** [play-with-docker/play-with-docker.github.io#67]
On the http://birthday.play-with-docker.com/webapps/ page, the below text isn't clear:

```text
Note: The current version of this image doesn’t run without the -d flag. The -d flag enables detached 
mode, which detaches the running container from the terminal/shell and returns your prompt after the
container starts. We are debugging the problem with this image but for now, use -d even for this first
example.
```

Particularly the last sentence `We are debugging the problem with this image but for now, use -d even for this first example.` isn't obvious.

**Comment by marcosnils** [comment]
@neomatrix369 good catch. I guess we can remove that warning totally. I believe it's unnecessary. @ManoMarks ?

**Comment by neomatrix369** [comment]
Once confirmed, please assign the task to me I'll fix it

**Comment by ManoMarks** [comment]
Fixed in #73
